src/components/shared/forms/pill.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Svg } from 'components/shared/svg/svg';
+import { joinClasses } from 'utilities';
+
+import './styles/pill.css';
+
+export const Pill = (props) => {
+  const { svg, label, onSvgClick} = props;
+  return (
+    <div className={joinClasses('pill', 'pill-primary')} onClick={onSvgClick}>
+      <p>{label}</p>
```

Comment:
Instead of using vanilla `p` and `div` tags these items should be positioned with flex-box applied with classnames. Checkout the btn component for an example of the styling.

Reply:
Changed.